src/lib/viewers/ActionSummaryViewer.ts

```diff
@@ -29,7 +38,9 @@ export class ConfigActionSummaryViewer extends AbstractActionSummaryViewer {
 		super(display);
 	}
 
-	public view(logFile: string, outfile?: string): void {
+	public viewPostExecutionSummary(logFile: string, outfile?: string): void {
```

Comment:
The concrete summary viewers' `view` method was renamed to `viewPostExecutionSummary` to distinguish it from the net-new `viewPreExecutionSummary`.